cpp/src/io/parquet/compact_protocol_writer.hpp

```diff
@@ -100,6 +101,8 @@ class CompactProtocolFieldWriter {
 
   inline void field_struct_blob(int field, const std::vector<uint8_t>& val);
 
+  inline void field_binary(int field, const std::vector<uint8_t>& val);
```

Comment:
Please change this+above+below to east const.

Reply:
updated.